src/graphql/resolvers/queries/communities.js

```diff
@@ -60,5 +61,50 @@ export const fieldResolvers = {
         totalEvents: allEvents.length,
       };
     },
+    sendDigest: async (
+      { name },
+      { hours, start },
+      { dataSources: { firestore } },
+    ) => {
+      dlog('sendDialog called for %s, hours: %s', name, hours);
+      if (!hours) throw new Error('hours parameter required');
+      if (hours < 1) throw new Error('hours minimum value is 1');
+      if (hours > 168) throw new Error('hours maximum value is 168');
+      const activeEvents = await communityStore(firestore).findIsActiveEvents(
+        name,
+      );
+      let digestStart = 'CURRENT_HOUR';
+      if (start) digestStart = start;
+
+      let atDate;
+      // Date as of now min, sec, ms set to zero
+      if (digestStart === 'CURRENT_HOUR') {
+        atDate = new Date(new Date(Date.now()).setMinutes(0, 0, 0));
+      } else if (digestStart === 'NEXT_HOUR') {
+        // now + 1 hour (3600000 ms)
+        atDate = new Date(
+          new Date(Date.now()).setMinutes(0, 0, 0) + 60 * 60 * 1000,
```

Comment:
I pretty sure `new Date()` is the same as `new Date(Date.now())`

Reply:
I just tested in REPL, it is not the same.

Reply:
so I did this

```
const x = new Date();
const y = new Date(Date.now());
```

I got the same result.

Reply:
Yeah, my error, I wasn't in the REPL

Reply:
new dates are hard. common core.

Reply:
fixed in commit 05d7e4d